source/elements/oneDAL/source/terminology.rst

```diff
@@ -0,0 +1,64 @@
+.. _terminology:
+
+============
+Terminology
+============
+
+@table-begin:
+   column-sizes: [ 25% ]
+Term | Definition \\
+
+Batch Mode | The computation mode for an algorithm in oneDAL, where all the data needed for
+computation are available at the start and fits the memory of the device where computations will
+be performed. \\
+
+Dataset | A pair of a matrix $X$ of size $N \times p$ and a vector $Y$ of size $N$, where $X$
+contains $N$ feature vectors in rows and $Y$ stores all corresponding responses. Vector $Y$ is
+optional and may not be present if responses are unknown. \\
+
+DPC++ | DPC++ --- Technology (programming language, compiler and runtime) that
+is based on SYCL specification and extends it. \\
```

Comment:
SYCL* (the first usage in a topic should be with an asterisk)

Reply:
fixed